browser/src/control/jsdialog/Widget.MenuButton.js

```diff
@@ -146,6 +146,8 @@ function _menubuttonControl (parentContainer, data, builder) {
 		if (data.image) {
 			var image = L.DomUtil.create('img', '', button);
 			image.src = data.image;
+			if (button.id.indexOf('color') !== -1)
+				image.setAttribute('alt', '');
```

Comment:
I think it should be the same as label of the icon: `data.text ? data.text : ''`
So the screen reader will get correct text.

Reply:
Unfortunately, the proposed solution, it is out of the scope of the task description. Please I advise you to change the task text description.